src/ctim/schemas/openc2_network.clj

```diff
@@ -0,0 +1,104 @@
+(ns ctim.schemas.network_coa
+  (:require [schema.core :as s]
+            [ctim.schemas.openc2vocabularies :as openc2v]))
+
+
+
+
+(s/defschema BGPBlackhole
+  {:type (s/enum "BGPBlackhole")
+   :host s/Str})
+
+(s/defschema DNSSinkhole
+  {:type (s/enum "DNSSinkhole")
+   :host s/Str})
+
+(def Protocol
+  (s/enum "TCP"
+          "UDP"
+          "ICMP"
+          "Any"))
+
+(def ACLAction
+  (s/enum "ALERT" 
+          "DROP"
+          "DENY"
+          "LOG"
+          "PASS"
+          "REJECT"))
+
+(s/defschema Traffic
+  {:protocol Protocol
+   :sourceAddress s/Str
+   :sourcePort s/Str
+   :destinationAddress s/Str
+   :destinationPort s/Str})
+
+(s/defschema NetworkACL
+  {:type (s/enum "NetworkACL")
+   :traffic Traffic
+   :action ACLAction})
+
+
+(s/def vlanProfile
+  {:vlanTag s/Str})
+
+(s/def secGroupProfile
+  {:secGroupTag s/Str
+   :secGroupACL s/Str})
+
+
+(s/defschema Remediation
+  {:type (s/enum "Remediation")
+   :server s/Str
+   :containmentProfile (s/either vlanProfile
+                                 secGroupProfile)
+   :acl NetworkACL})
+
+(s/defschema NonSensitive
+  {:type (s/enum "NonSensitive")
+   :permissibleIps [s/Str]
+   :acl NetworkACL})
+
+(s/defschema Honeypot
+  {:type (s/enum "Honeypot")
+   :permissibleIps [s/Str]
+   :acl NetworkACL 
+   :routes {:prefix s/Str
+            :NextHop s/Str
+            :NextHopType s/Str}})
+
+(def Encapsulation
+  (s/enum "GRE"
+          "VXLAN"))
+
+(s/defschema BlockModifier
+  {:type (s/enum 
+           "Perimeter"
+           "Internal")
+   :method (s/conditional
+            #(= "NetworkACL" (:type %)) NetworkACL
+            #(= "BGPBlackhole" (:type %)) BGPBlackhole
+            #(= "DNSSinkhole" (:type %)) DNSSinkhole)})
+
+
+  
+
+(s/defschema ContainModifier
+  {:type (s/eq "Contain")
+   :method (s/conditional
```

Comment:
The problem is not with swagger UI, but the swagger spec itself, this is not a bug, it's a choice from the developers on the spec.

Violating the spec not only breaks Swagger UI, but also code generation and makes our API incompatible with any standard Swagger tool.

there are many huge threads about this issue, mainly:
https://github.com/OAI/OpenAPI-Specification/issues/57

I was suggesting this change to know if there is any "smart decision" we could make on this schema to avoid writing a workaround, for instance isn't @craigbro previously proposed solution eligible here? instead of having:

```clojure
:method (s/conditional
             #(= "NetworkACL" (:type %)) NetworkACL
             #(= "BGPBlackhole" (:type %)) BGPBlackhole
             #(= "DNSSinkhole" (:type %)) DNSSinkhole)})
```

maybe we could use something like this:

```clojure
(st/optional-keys 
{:method_NetworkACL NetworkACL
 :method_BGPBlackhole BGPBlackhole
 :method_DNSSinkhole DNSSinkhole})
```